guides/common/modules/proc_installing-puppet-agent-during-host-provisioning.adoc

```diff
@@ -49,11 +53,14 @@ ifndef::katello,orcharhino,satellite[]
 +
 Select the *boolean* type and set the value to `true`.
 endif::[]
-ifdef::katello,orcharhino,satellite[]
-. Enable the Puppet agent using a host parameter in global parameters, a host group, or for a single host.
-Add a host parameter named `enable-puppet7`, select the *boolean* type, and set the value to `true`.
-endif::[]
+. If you deploy the Puppet agent for a non-integrated Puppet setup, configure the following host parameters in global parameters or a host group:
+* Add a host parameter named `puppet_server`, select the *string* type, and set the value to the hostname of your Puppet server, such as `puppet.example.com`.
+* Optional: Add a host parameter named `puppet_ca_server`, select the *string* type, and set the value to the hostname of your Puppet CA server, such as `puppet-ca.example.com`.
+If `puppet_ca_server` is not set, the Puppet agent will use the same server as `puppet_server`.
+* Optional: Add a host parameter named `puppet_environment`, select the *string* type, and set the value to the Puppet environment you want the host to use.
+
++
+If you use an integrated setup, ensure that you select a Puppet {SmartProxy}, Puppet CA {SmartProxy}, and Puppet environment when you create your host.
```

Comment:
@adamlazik1 Does this make it clearer?

Reply:
Maybe I understood it wrong, does selecting the values when creating the host not refer to this procedure but is a different, non-related procedure?

Reply:
Yes, it's a different step. One of the two must be done, but they're done under different circumstances and in a different order in relation to the other steps in this module.

Reply:
I see, thanks for clarifying.